src/openforms/js/components/admin/form_design/variables/ServiceFetchConfigurationForm.js

```diff
@@ -62,6 +62,27 @@ const ServiceFetchConfigurationForm = ({formik, selectExisting = false}) => {
             }
             extraClassName="admin-fieldset"
           >
+            <FormRow>
+              <Field
+                name={'name'}
+                fieldBox
+                required
+                label={
+                  <FormattedMessage
+                    defaultMessage="Name"
+                    description="Service fetch configuration modal form name field label"
+                  />
+                }
+              >
+                <TextInput
+                  id="name"
+                  value={formik.values.name}
+                  maxLength="1000"
+                  {...formik.getFieldProps('name')}
+                />
```

Comment:
```suggestion
                <TextInput
                  id="name"
                  maxLength="1000"
                  {...formik.getFieldProps('name')}
                />
```
the `value` prop should already be included with `formik.getFieldProps`

Even cleaner would be:

```jsx
import {Field as FormikField} from 'formik';

<FormikField name="name" as={TextInput} id="name" maxLength="1000" />
```

The `as` prop allows specifying your own components, while any additional props are forwarded to the `as`-component. The `name` is taken and under the hood formik grabs all the relevant props.

Reply:
That `FormikField` seems to need some `Formik` component and a `FormikContext`. I went with just removing the `value` attribute.

Reply:
ah right, this works differently then because this case uses the `useFormik` hook.

I don't like that formik offers three ways to do the same thing and that we now use all three smeared out over this repo, the SDK repo and the formio-builder/formio-renderer repo's :thinking: 

I'll keep it in the back of my head to clean up at some point and review more strictly. Removing the value attribute is sufficient for this PR :+1: